### DISCUSSION TITLE
Tried installing rustdesk-server-s6, but there's no web console?

### DISCUSSION THREAD

**Discussion by cwm9cwm9** [rustdesk/rustdesk-server#312] [Q&A]
_Edit for those that come after me:_

_The non-pro version does not have the web console, and that's why there isn't one in the S6 container.  There is a separate process for installing the Pro version via Docker that does not involve these containers._

Hi, I'm new to rustdesk.

I installed the rustdesk-server-s6 package, and the instructions are to forward ports 21115-21119/tcp and 21116/udp... and that leaves out 21114.  I tired to connect to each of the ports I forwarded with my browser, but I just got denied each time.

So... is there no web console at all in the s6 container?  How are you supposed to configure the server?

Thanks.

**Comment by rustdesk** [comment]
https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/
